build/webpack.styles.js

```diff
@@ -1,27 +1,39 @@
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
+// import postcssPlugins from './webpack.postcss'
 
-export default {
-  dev: {
+const styleLoaders = {
+  development: {
     test: /\.css$/,
     loaders: [
       'style-loader',
-      { loader: 'css-loader', query: { modules: true } },
-      { loader: 'postcss-loader' }
+      { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
+      { loader: 'postcss-loader'
+        // pending new postcss-loader release - then remove loaderOptionsPlugin
+        // options: {
+        //   plugins: function () {
+        //     return postcssPlugins
+        //   }
+        // }
+      }
     ]
   },
-  prod: {
+  production: {
     test: /\.css$/,
-    loader: ExtractTextPlugin.extract({
+    loaders: ExtractTextPlugin.extract({
       fallbackLoader: 'style-loader',
       loader: [
-        { loader: 'css-loader', query: { modules: true } },
-        { loader: 'postcss-loader' }
+        { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
+        { loader: 'postcss-loader'
```

Comment:
prod = extractTextPlugin, dev = not.

I agree with pulling a bit of it out into a var so it can be shared...but there is subtle syntax differences - especially with the updates to ExtractTextPlugin for webpack2